.travis.yml

```diff
@@ -2,8 +2,7 @@ language: go
 dist: trusty
 
 go:
-  - 1.9.x
```

Comment:
Could we add `1.11.x` as well? Just to be sure that it's working.

Reply:
I readded all previous versions and added 1.12.

Reply:
And now 1.11, too